forms/user/registration_form_test.go

```diff
@@ -0,0 +1,393 @@
+package forms_test
+
+import (
```

Comment:
Please order the packages in the following order
```go
import (
  // Standard library packages

  // Internal project packages

  // Third-party packages
)

```

Reply:
😵